src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -357,6 +368,45 @@ class CacheGetResponseError extends CacheGetResponse
     use ErrorBody;
 }
 
+abstract class CacheSetIfNotExistsResponse extends ResponseBase
+{
+    public function asStored(): CacheSetIfNotExistsResponseStored|null
+    {
+        if ($this->isStored()) {
+            return $this;
+        }
+        return null;
+    }
+
+    public function asNotStored(): CacheSetIfNotExistsResponseNotStored|null
+    {
+        if($this->isNotStored()) {
+            return $this;
+        }
+        return null;
+    }
+
+    public function asError(): CacheSetIfNotExistsResponseError|null
+    {
+        if ($this->isError()) {
+            return $this;
+        }
+        return null;
+    }
+}
+
+class CacheSetIfNotExistsResponseStored extends CacheSetIfNotExistsResponse {
+}
```

Comment:
Our vanilla `set` operatio returns the key and value in the response object. Is there any value in having `CacheSetIfNotExistsResponseStored` behave the same way?

Reply:
Sure, I can do that for consistency.

Reply:
Done in [e454276](https://github.com/momentohq/client-sdk-php/pull/91/commits/e45427622a5cad451d229776518230f4e62e5e6f)